index.bs

```diff
@@ -567,20 +570,15 @@ to provide a correct bearing.
 <dfn>Smart sensors</dfn> and <dfn>sensor hubs</dfn>
 have built-in compute resources which allow them
 to carry out [=calibration=] and [=sensor fusion=] at the hardware level,
-freeing up CPU resources
-and lowering battery consumption
-in the process.
+freeing up CPU resources and lowering battery consumption in the process.
 
-But [=sensor fusion=] can also be carried out in software.
+[=Sensor fusion=] can also be carried out in software.
 This is particularly useful when performance requirements can only be met
 by relying on application-specific data.
 For example, head tracking for virtual or augmented reality applications
-requires extremely low latency
-to avoid causing motion sickness.
-That low-latency is best provided
-by using the raw output of a gyroscope
-and waiting for quick rotational movements of the head
-to compensate for drift.
+requires extremely low latency to avoid causing motion sickness.
+That low-latency is best provided by using the raw output of a gyroscope
```

Comment:
I'm not sure whether this statement is valid. What if sensor does fusion in hw  + predictive tracking algorithm? It would be better than raw gyroscope. Also, it is unclear how quick rotational head movements compensate drift, usually, magnetometer is used to compensate drift, or in worst case, accelerometer.

Reply:
here I did not modify the text. I agree that this statement is not obvious, so we should either provide a confirming link or drop this paragraph, wdyt?

Reply:
Yes, I would propose to drop this paragraph and add motion-to-photon (MTP) latency example in previous one (to clarify `extremely`). There are quite many research papers in this area, e.g., lag over 20ms considered noticeable and can cause motion sickness.